docs(Self-Managed): Backport troubleshooting (#7400)
camunda/camunda-docs: versioned_docs/version-8.8/self-managed/deployment/helm/upgrade/helm-870-880.md

## Troubleshooting

### Migration job did not complete

If the migration job (for data or identity) fails to complete and eventually stops retrying, it likely hit its `backoffLimit` (the maximum number of retry attempts).

> This usually happens when the importer job wasn’t finsihing yet before the jobs stops.

Reviewer: ```suggestion
This typically happens if the importer job did not complete before the job stopped.
```
